open-sphere-base/control-panels/src/main/java/io/opensphere/controlpanels/SimpleRegistry.java

```diff
@@ -8,23 +8,29 @@
 /**
  * A registry of detail panel providers. Plugins may register a provider with
  * the registry to add additional behavior.
+ * 
+ * @param <T>
  */
-public interface DetailPanelProviderRegistry
+public interface SimpleRegistry<T>
 {
     /**
      * Registers the supplied provider with the registry.
+     * 
+     * @param <T>
```

Comment:
You shouldn't need this parameter documentation, it's documented at the class level.